tests/test_crew_ai/test_library_context.py

```
@@ -0,0 +1,210 @@
+        assert isinstance(ctx, BrowserLibraryContext)
+
+    def test_case_insensitive_selenium(self):
+        ctx = get_library_context("SELENIUM")
+        assert isinstance(ctx, SeleniumLibraryContext)
+
+    def test_unknown_type_raises_value_error(self):
+        with pytest.raises(ValueError, match="Unknown library type"):
+            get_library_context("playwright")
+
+    def test_empty_string_raises_value_error(self):
+        with pytest.raises(ValueError):
```

Ruff (0.15.7)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)